openedx/openedx-platform
# fix: fixing the prompt view cancel button style for color and cursor shape

## Description

This fixes the Prompt View `Cancel` option to have the proper style for color and cursor shape. 

This was a style that missed when the element for `action-secondary` was updated from `button` to `a` here:
https://github.com/openedx/edx-platform/commit/3eb4e0ce9cebabea5c3162dc761bf462ea6177f5#diff-0cc73d38a75fb2b57e41c9660dec6dad6fd079d8b8322b19d369e5baaadc3c52L33-R33

The change follows the style that was implemented for other `action-secondary` elements:
https://github.com/openedx/edx-platform/commit/3eb4e0ce9cebabea5c3162dc761bf462ea6177f5#diff-a21e3eaa1e33ca8a10c73f3a53dca3d58ee463ddfe1eb94a9574e8762396c4ceR586-R594

## Supporting information

* fixes https://github.com/openedx/wg-build-test-release/issues/322

## Testing instructions

1. Set up tutor nightly (steps can be found here: https://docs.tutor.edly.io/tutorials/nightly.html).
2. Create an admin user: `tutor dev do createuser --staff --superuser admin admin@test.com`
3. Go to studio: http://studio.local.overhang.io:8001/
4. Create a course.
5. Click into the course.
6. Go to Course Updates by clicking on `Content` in the top menu, then on `Updates`.
7. Create an update by clicking on `+ New update` in the top right.
8. Click on the `Delete` button for the update to see the confirmation prompt.
9. Observe the color and hover the `Cancel` button to reproduce the problem described in the issue (https://github.com/openedx/wg-build-test-release/issues/322).
10. Update to use this branch by mounting a clone of the repository (steps can be found here: https://docs.tutor.edly.io/dev.html)
11. Rebuild.
12. Refresh the Course Updates page.
13. Click on the `Delete` button for the update to see the confirmation prompt.
14. Confirm that the color and cursor for the delete link are fixed.

## Deadline

N/A

## Other information

This is a video of the functionality from the `palm` release to show the expected style and behavior:
![testing_bug_322_incorrect_cursor_icon_in_palm_release](https://github.com/openedx/edx-platform/assets/8575685/d256fd88-73ca-400b-b2f3-bc82ce297f7a)

This is a video reproducing the issue:
![testing_bug_322_reproduce](https://github.com/openedx/edx-platform/assets/8575685/5597a48d-687f-4aa7-b8c3-78fb75ec3d33)

This is a video of the fix:
![testing_bug_322_fix](https://github.com/openedx/edx-platform/assets/8575685/45714dac-8992-4539-aa12-01395cb6d649)

The white color for this was chosen because it was approved for a similar element here (note the orange button, orange header line, and white cancel link):
https://github.com/openedx/edx-platform/pull/33197#issuecomment-1719175975
